src/touchstone/databases/elasticsearch.py

```diff
@@ -64,7 +64,7 @@ def emit_compute_dict(self, uuid, compute_map, index, identifier):
                 f"Incorrect JSON data: nested dictionaries aggregations \
 fields are required in {compute_map}"
             )
-            exit(1)
+            return output_dict
```

Comment:
exit(1) means there's an issue in the configuration file, why do we want to continue the execution after an issue like this?

Reply:
imho a lib shouldn't exit, but pass the error up to the caller. vs exit.